Add Rafter Katacoda tutorials (#40)
kyma-project/rafter: docs/katacoda-tutorials/step4.md

2. Apply an Asset CR that points to a single [AsyncAPI](https://asyncapi.org/) specification file. To validate the specification and convert it to the latest 2.0 version before it goes into storage, in the Asset CR specify communication with **validationWebhookService** and **mutationWebhookService**. The webhook services are separate services responsible for the validation and conversion of AsyncAPI specifications. As a result, the file in storage will be an AsyncAPI file in 2.0 version.

   ```yaml
   cat <<EOF | kubectl apply -f -
   apiVersion: rafter.kyma-project.io/v1beta1
   kind: Asset
   metadata:
     name: asyncapi-file
     namespace: default
   spec:
     source:
       url: ${ASYNCAPI_FILE_URL}
       mode: single
       validationWebhookService:
         - name: rafter-rafter-asyncapi-service
           namespace: default
           endpoint: "/v1/validate"
       mutationWebhookService:
         - name: rafter-rafter-asyncapi-service
           namespace: default
           endpoint: "/v1/convert"
     bucketRef:
       name: content
   EOF
   ```{{execute}}

> 3. Check if the Asset status is `Ready` which means that fetching and communication with the AsyncAPI Service is completed. Run:

Reviewer: ```suggestion
3. Check if the Asset status is `Ready` which means that fetching and communication with the AsyncAPI Service was completed. Run:
```

   `kubectl get assets asyncapi-file -o jsonpath='{.status.phase}'`{{execute}}

> To make sure that the file is in storage and you can extract it, proceed with the next two steps:

Reviewer: ```suggestion
To make sure that the file is in storage and you can extract it, follow these steps:
```
